src/serverconf/src/main/java/ee/ria/xroad/common/conf/serverconf/ServerConfImpl.java

```diff
@@ -73,6 +73,15 @@
 
     private static SecurityServerId identifier;
 
+    /**
+     * Initializes a new ServerConfImpl object.
+     */
+    public ServerConfImpl() {
+        synchronized (this) {
```

Comment:
Synchronizing on `this` does not make sense in a constructor (see e.g. https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.8.3). If the intention is to protect the static field from concurrent modification, the synchronization should be on the class object (ServerConfImpl#getIdentifier has the same mistake). 
Why is the `identifier` field static in the first place? It seems that it could be a regular instance variable, which would make this constructor unnecessary (and make the lazy initialization in getIdentifier correct).

Reply:
The `identifier` field was static originally and was not changed in this task. Probably the original reason for that was that the owner could never change. It was changed to a regular instance variable in the commit 330bfaa.